modules/newFunc.js

```diff
@@ -0,0 +1,88 @@
+const tasksSection = document.querySelector('.task-section');
+const taskDesc = document.getElementById('task-value');
+
+let tasks = [
+  {
+    description: 'Create social Link',
+    completed: false,
+    index: 10,
+  }, {
+    description: 'Do Coding Exercise',
+    completed: false,
+    index: 11,
+  }, {
+    description: 'Doing physical Exercise',
+    completed: false,
+    index: 12,
+  },
```

Comment:
- [ ] [optional] Firstly I apologize for this change requested by the previous reviewer, It was not part of the requirements. I would suggest removing these hardcoded tasks from the array It makes a bad experience for users. You can skip this change it will not block your project from approval.